libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -224,8 +224,12 @@ bool AudioIO::ValidateDeviceNames(const wxString &play, const wxString &rec)
    return pInfo != nullptr && rInfo != nullptr && pInfo->hostApi == rInfo->hostApi;
 }
 
+static constexpr auto kTimerInterval = std::chrono::milliseconds{ 50 };
+
 AudioIO::AudioIO()
 {
+   wxTimer::Start(kTimerInterval.count());
```

Comment:
There is no safety around what kind of duration the kTimerInterval will be. Explicit `duration_cast<milliseconds>` would help.